test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConditionalCreateTests.cs

```diff
@@ -104,5 +106,35 @@ public async Task GivenAResource_WhenCreatingConditionallyWithEmptyIfNoneHeader_
 
             Assert.Equal(HttpStatusCode.BadRequest, exception.Response.StatusCode);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenConcurrentResources_WhenCreatingConditionallyWithNoIdAndNoExisting_TheServerShouldReturnOneResourceSuccessfully()
```

Comment:
Not sure if it is possible to create a test which replicates the resources locked case and we see the resource conflict result?

Reply:
When I run this locally I see 1 created and 19 conflicted. I added `|| x == HttpStatusCode.OK` which I believe is also valid where the initial request finished before others had started.

Given the attempt to run 20 requests at once, I'm not sure how to isolate any further to guarantee Conflicts.